app/assets/stylesheets/application.scss

```
@@ -183,4 +183,37 @@ form.search-form input.search-text {
   font-size: 1.5rem !important;
   height: auto !important;
   padding-left: 50px;
+}
+
+//テキスト教材詳細ページ
+pre {
+  border: 0.1px solid;
+  padding: 10px;
+}
+
+#text-contents table {
+  border-collapse: collapse;
+  border: 1px solid #ccc;
+  line-height: 1.5;
+  padding: 10px;
+  font-size: 1.4em;
+  margin: 30px auto;
+}
+
+thead {
+  display: table-header-group;
+  vertical-align: middle;
+  border-color: inherit;
+}
+
+tr {
+  display: table-row;
+  vertical-align: inherit;
+  border-color: inherit;
+}
```

このような感じにすればマークダウンのテーブルにも対応できます。

```suggestion
// マークダウン

section#text-contents table {
  @extend .table;
  @extend .table-striped;
}
```

---

preを消すと囲み枠も消えたので残しておきました